lib/ReactViews/Custom/Chart/Legends.jsx

```diff
@@ -21,10 +22,7 @@ export default class Legends extends React.PureComponent {
     });
 
     return (
-      <div
-        className={Styles.legends}
-        style={{ maxHeight: `${Legends.maxHeightPx}px` }}
-      >
+      <Container style={{ maxWidth: this.props.width }}>
```

Comment:
is there a reason we aren't using our <Box> component? I'm a bit wary of introducing another general purpose container component

Reply:
Furthermore, is there a reason why we are giving this div a max width?

Reply:
Ah, that's a local container defined here, not a generic or exported one. I have now renamed it as `LegendsContainer` for clarity.

https://github.com/TerriaJS/terriajs/blob/947e040504bfe05e7bc48d12d27eb070c4060811/lib/ReactViews/Custom/Chart/Legends.jsx#L68

We need `max-width` to constraint the legends div to the size of the plot. Without max-width (and I think because we use white-space: nowrap) the div expands to fit the content which can exceed the whole page width.